mesa/experimental/jupyter_viz.py

```diff
@@ -71,6 +71,7 @@ def JupyterViz(
             simulations with no space to visualize should
             specify `space_drawer=False`
         play_interval: play interval (default: 150)
+        seed: the random seed used to initialize the model
```

Comment:
Can we explicitly pass in the random model seed so it is obvious to the user where the seed is coming form and what they are changing is they pass in their own?

Reply:
This is already an explicit argument. Specified by adding a "seed" entry to the `model_params` dictionary: https://github.com/projectmesa/mesa-examples/blob/main/examples/schelling_experimental/app.py#L21.

Reply:
I forgot to remove that line from the docstring.

Reply:
> Scenario 1: User does not specify the seed so we want the UI to show the seed that was selected. Shouldn't the display of the random seed reference [obj._seed](https://github.com/projectmesa/mesa/blob/8a50f9f36392b95824e0e2768dcf6f16515514d8/mesa/model.py#L58)

The direction of the cause and effect should flow from the controls you see on the screen, and then the model properties. It can't be the `obj._seed` to have a value before the control.

If you notice, the seed value is displayed twice. The one below the information card is `model._seed`, displayed for debugging purpose. I will remove this once you agree to merge.